starter_project/autonomy/msg/StarterProjectTag.msg

```diff
@@ -0,0 +1,4 @@
+int32 tagId
```

Comment:
we should remove all these starter project files right

Reply:
Fixed?

Reply:
Looks like it. Instead of doing this manually (idk what you did) you can do `git restore --source origin/integration starter_project` or whatever to revert whole folders to certain branches (in this case the target merge branch).